.github/workflows/build.yaml

```
@@ -0,0 +1,96 @@
+on:
+  workflow_call:
+    outputs:
+      artefact:
+        description: The name of the uploaded image aretfact.
+        value: ${{ jobs.build.outputs.artefact }}
```

```suggestion
      artifact:
        description: The name of the uploaded image artifact.
        value: ${{ jobs.build.outputs.artifact }}
```

These are more commonly called `artifacts`, [here's an ngram](https://books.google.com/ngrams/graph?content=artefact%2Cartifact&year_start=1900&year_end=2000&corpus=26&smoothing=3). It's also called artifact [by GitHub documentation](https://docs.github.com/en/actions/advanced-guides/storing-workflow-data-as-artifacts), so even though `artefact` is etymologically justifiable, I think it's probably cleaner for us to go with `artifact`.

And yes I realize this is exceptionally pedantic of me 😁 

There are a bunch of these further down, but I'll leave it up to you whether you want to make this change or not, and if you're gonna do it, probably do it with a search and replace.

---

Renamed in 6b3ec28